index.tt

```diff
@@ -6,152 +6,349 @@
 
 <link rel="alternate" type="application/rss+xml" title="RSS" href="/news-rss.xml" />
 
-<div class="nixos-org-index">
-  [%# Title for the site, used to keep the document tree rooted at h1. %]
-  <h1>NixOS</h1>
-  
-  <div class="jumbotron-two"><div>
-    <div class="intro-nix">
-      <h2>What is Nix?</h2>
-    
-      <p>
-      Nix is a powerful package manager for Linux and other Unix systems
-      that makes package management reliable and reproducible.  It
-      provides atomic upgrades and rollbacks, side-by-side installation
-      of multiple versions of a package, multi-user package management
-      and easy setup of build environments.
-      </p>
-    
-      <div class="learn-more">
-        <a class="btn btn-success" href="[%root%]nix/">
-          <i class="fa fa-chevron-right"></i> Learn more about Nix
-        </a>
+<div class="row-fluid">
+
+  <div class="span6">
+    <h1>Reproducible builds and deployments.</h1>
+    <p class="lead">
+      NixOS, an open source software, gives you tools to share
+      your development and build environments between machines with confidence.
+      Many know NixOS as the <strong>declarative</strong> Linux distribution
```

Comment:
>Many know

could be understood as if it were not true. why not "NixOS is configured declaratively"? yes declarative would be nicer as bold word

you are in the right direction!

Reply:
closing since i changed it already